server/controllers/manage/lostBedsController.ts

```diff
@@ -127,4 +127,21 @@ export default class LostBedsController {
       }
     }
   }
+
+  cancel(): RequestHandler {
+    return async (req: Request, res: Response) => {
+      const { premisesId, bedId, id } = req.params
+      const { notes = '' } = req.body
+
+      try {
+        await this.lostBedService.cancelLostBed(req.user.token, id, premisesId, { notes })
+
+        req.flash('success', 'Bed cancelled')
+
+        return res.redirect(paths.lostBeds.index({ premisesId }))
+      } catch (err) {
+        return catchValidationErrorOrPropogate(req, res, err, paths.lostBeds.show({ premisesId, bedId, id }))
```

Comment:
do we want to redirect to the show lost bed path, or stay on the edit screen?

Reply:
Yeah good shout, now I've made the change Stu suggested they will be directed to the index